.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,26 +2,26 @@ Issue: #
 <!-- Link to relevant issue. All PRs should be asociated with an issue -->
 
 ## PR Type
-What kind of change does this PR introduce?
 
+What kind of change does this PR introduce?
 <!-- Please check the one that applies to this PR using "x". -->
-```
-[ ] Bugfix
-[ ] Feature
-[ ] Code style update (formatting)
-[ ] Refactoring (no functional changes, no api changes)
-[ ] Build or CI related changes
-[ ] Documentation content changes
-[ ] Sample app changes
-[ ] Other... Please describe:
-```
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build or CI related changes
+- [ ] Documentation content changes
+- [ ] Sample app changes
+- [ ] Other... Please describe:
```

Comment:
Since PR Type will not change I think we should change this to bullets and ask devs to remove unnecessary types.

For example, If I submit a PR with Bugfix and Feature, the below format look good

## PR Type

* Bugfix
* Feature

when compared to the below format

## PR Type

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build or CI related changes
- [ ] Documentation content changes
- [ ] Sample app changes
- [ ] Other... Please describe:

Reply:
Agree, make each line as a comment and we can ask the submitter to uncomment the right line